{{cookiecutter.project_slug}}/.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## What this does
+
+Add a description of what feature/page this PR intends to add or change. Don't forget to start the PR title with the Jira ticket number.
+
+- [Jira ticket](Paste Jira link here) 
+
+## Checklist
+- [ ] Todo 1
+- [ ] Todo 2
+- [ ] Todo 3
+
+
+## Deploy Notes
+
+Add any notes for changes needed when deployed. I.e. API keys, environment variables, database changes, etc.
+
+## How to test
+
+Add user steps to achieve desired functionality for this feature.
```

Comment:
Maybe clean this up a little, but this is the same one that is currently added to generated projects.